docs/source/user_guide/automl.ipynb

```diff
@@ -61,7 +61,7 @@
    "source": [
     "The AutoML search will log its progress, reporting each pipeline and parameter set evaluated during the search.\n",
     "\n",
-    "By default, AutoML will search a fixed number of pipeline and parameter pairs (5). The first pipeline to be evaluated will always be a baseline model representing a trivial solution."
+    "There are a number of mechanisms to control the AutoML search time. One way is to set the maximum number of iterations using `max_iterations`. By default, AutoML will search a fixed number of iterations and parameter pairs (`max_iterations=5`). The first pipeline to be evaluated will always be a baseline model representing a trivial solution. "
```

Comment:
Suggest replace "One way is to set the maximum number of iterations" with "One way is to set the maximum number of candidate models to be evaluated during AutoML"